voyager/plugins/event-source/movement/src/main.rs

```diff
@@ -337,6 +338,10 @@ impl PluginServer<ModuleCall, ModuleCallback> for Module {
                             "TimeoutPacket" => from_raw_event::<ibc::TimeoutPacket>(data),
                             unknown => panic!("unknown event `{unknown}`"),
                         };
+
+                        // Log movement events
+                        info!(event_type = %event_type, tx_hash = %hash, height = %height, "Movement event detected");
```

Comment:
```suggestion
                        info!(event_type = %event_type, tx_hash = %hash, height = %height, "raw movement event");
```

Reply:
(this is more consistent with the log messages throughout the rest of our codebase)